packages/swi-prolog/build.sh

```diff
@@ -6,10 +6,10 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=9.1.0
 TERMUX_PKG_SRCURL=https://www.swi-prolog.org/download/devel/src/swipl-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=4718c426c58d09e76bd84c109ae59f47b8cc261213a47796296b3733f455a195
-TERMUX_PKG_DEPENDS="libarchive, libcrypt, libgmp, libyaml, ncurses, openssl, ossp-uuid, readline, zlib"
+TERMUX_PKG_DEPENDS="libarchive, libcrypt, libgmp, libyaml, ncurses, openssl, ossp-uuid, readline, zlib, pcre2"
 TERMUX_PKG_FORCE_CMAKE=true
 TERMUX_PKG_HOSTBUILD=true
-
+TERMUX_PKG_REVISION=1
```

Comment:
Please place `TERMUX_PKG_REVISION` adjacent to `TERMUX_PKG_VERSION` for maintainability.